apps/pie-microsite/src/utilities/helpers.js

```diff
@@ -32,8 +32,9 @@ const capitaliseFirstLetter = string => {
 };
 
 /**
- * Checks whether string can convert to number
+ * Checks whether string can convert to number - returns number if number is passed through
  * @param {string} string
+ * @param {number} number
```

Comment:
sorry - needs to be like this:
`* @param {string|number} value The value to test can be parsed as a number.`

(or something like that)